doc/source/administrator/optimization.md

```diff
@@ -383,3 +383,141 @@ Choosing the right _limit to guarantee ratio_ ratio is an art, not a science.
 We suggest **starting with a ratio of 2 to 1** and adjusting from there based on whether you run into problems on your hub.
 For example, if the limit is 10GB, start with a guarantee of 5GB.
 Use a service such as Prometheus + Grafana to monitor the memory usage over time, and use this to decide whether to adjust your ratio.
+
+## Explicit memory and CPU allocated to core pods' containers
+
+The Helm chart creates several k8s Pods that typically have one container
```

Comment:
Probably worth linking to k8s own [docs on the matter](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/) somewhere.

Reply:
Done